TeamShiksha/logoexecutive
# Create cft_dev_test_logoexecutive.yml

### Summary

Added `cloudformation` template for dev, and same can be used for test environment. The resources will be created as soon as it's upload in `cloudformation` service of AWS.

### Description

Closes #109 
1. Creates the resources on upload
2. Deletes the resources on deleting the stack(services storing some data like S3 will only be deleted if the s3 is empty)
3. Easy for everyone to created and delete resources.
4. resources include, `s3`, 'CloudFrontCDN', `User`

### How to Test the Changes

Upload the templates on AWS, and check the resources section of stack, to view of the resources which has been created by the template.

## Checklist
